systers_portal/meetup/views.py

```diff
@@ -19,3 +20,18 @@ class MeetupLocationList(ListView):
     template_name = "meetup/list_location.html"
     model = MeetupLocation
     paginate_by = 20
+
+
+class MeetupView(MeetupLocationMixin, DetailView):
```

Comment:
Thanks for suggestions :)

First approach mentioned above is working fine with just changing method `filter()` to `get()` because we are using `get_context_data` which make total sense.

Second approach is giving error - `'function' object has no attribute 'all'`. Also it is not showing for which line the error occurred.